bot/exts/filters/filtering.py

```diff
@@ -423,9 +424,12 @@ async def _send_log(
             # Allow specific filters to override ping_everyone
             ping_everyone = Filter.ping_everyone and _filter.get("ping_everyone", True)
 
-        # If we are going to autoban, we don't want to ping
+        content = str(msg.author.id)  # quality-of-life improvement for mobile moderators
+
+        # If we are going to autoban, we don't want to ping and don't need the user ID
         if reason and "[autoban]" in reason:
             ping_everyone = False
+            content = ""
```

Comment:
To be explicit.
```suggestion
            content = None
```

Reply:
https://github.com/python-discord/bot/pull/2098/commits/6c561e251be3a4af2af349ba4b86b9ff0f0a9b7b